new_app_setup.py

```diff
@@ -0,0 +1,266 @@
+
+    config_file = get_congif_yml_file(team_name)
+
+    path_to_congif_file = os.path.join(base_path, config_file)
+
+    send_config_email_to_users(team_name, args.group_user, config_file)
+
+    if args.list_of_users is not None:
+        send_config_email_to_users(team_name, args.list_of_users, config_file)
+
+
+def create_kubernetes_namespace_yml(namespace_filename, team_name):
+
+    tag = "Hack_For_A_Cause_2019"
+
+    yml_file_kubernetes_data = dict(
+
+        kind='Namespace',
+        apiVersion='v1',
+        metadata=dict(
+            name=team_name,
+            labels=dict(
+                name=tag
+                )
+            )
+        )
+
+    with open(namespace_filename, 'w') as outfile:
+        yaml.dump(yml_file_kubernetes_data, outfile, default_flow_style=False)
+
+    print("The YAML file for {} was created and was "
+          "saved as {}".format(team_name, namespace_filename))
+
+    return yml_file_kubernetes_data
+
+
+def create_kubernetes_pods_yml(pods_filename, team_name, ram_requested,
+                               cpu_requested):
+    yml_file_kubernetes_data = dict(
+        kind='Pod',
+        apiVersion='v1',
+        metadata=dict(
+            name=team_name,
+            namespace=team_name,
+            ),
+        spec=dict(
+                containers=dict(
+                    name=team_name,  # QUESTION HERE -> see below
+                    resources=dict(
+                        limits=dict(
+                            cpu="1",
+                            memory="2G"
+                        ),
+                        requests=dict(
+                            cpu=cpu_requested,
+                            memory=ram_requested,
+                        )
+                    )
+                )
+            )
+        )
+# QUESTION: When running, I am getting the warning:
+
+# "error: error validating "test-team-name_pods.yml": error validating
+# data: ValidationError(Pod.spec.containers): invalid type for
+# io.k8s.api.core.v1.PodSpec.containers: got "map", expected "array";
+# if you choose to ignore these errors, turn validation off with
+# --validate=false
+# and I know that is because name does not have the hyphen indicating an array
+# in front of it, but I cannot seem to figure out how to do this...
+# anything I'm missing?
+
+    with open(pods_filename, 'w') as outfile:
+        yaml.dump(yml_file_kubernetes_data, outfile, default_flow_style=False)
+
+    print("The YAML file for {} pods were created and was "
+          "saved as {}".format(team_name, pods_filename))
+
+    return yml_file_kubernetes_data
+
+
+def create_kubernetes(filename, team_name, type_of_create):
+    subprocess.call(['kubectl', 'create', '-f', filename])
+
+    print("The Kubernetes {} for {} was "
+          "created.".format(type_of_create, team_name))
+
+
+def create_bucket(team_name):
+    pass
+    # export PROJECT_ID=$(gcloud config get-value team_name)
```

Comment:
our project_id is mvpstudio this could be read in through a command line argument

Reply:
We want to create 2 buckets, one for file storage and another for database backups. Something like, 
`statg-projectname-files` & `stage-projectname-db-backups`